src/util/fees.cpp

```diff
@@ -31,6 +31,7 @@ bool FeeModeFromString(const std::string& mode_string, FeeEstimateMode& fee_esti
         {"UNSET", FeeEstimateMode::UNSET},
         {"ECONOMICAL", FeeEstimateMode::ECONOMICAL},
         {"CONSERVATIVE", FeeEstimateMode::CONSERVATIVE},
+        {"btc/kb", FeeEstimateMode::BTC_KB},
```

Comment:
I think we use upper case. See also

```
src/policy/feerate.cpp:const std::string CURRENCY_UNIT = "BTC";
```

which you can use to express your love of shitcoins.

Reply:
I am very tempted to make the parser case insensitive and use CURRENCY_UNIT as you say. Otherwise it's a bit of a pain to require users to type "BTC/kb" every time (as opposed to just "btc/kb")

Reply:
Also, there is a difference between kilobit (kb) and kilobyte (kB)

Reply:
+1 for case insensitive

Also fine with CURRENCY_UNIT

I don't care about the `kB` vs `kb` distinction as far as the parser goes, but documentation wise we should do it correctly. Users will out by themselves it's case insensitive :-)

Reply:
RPC is for software, not users...

Reply:
It's for any entity that doesn't use the GUI, which includes non-software me.